src/_data/side-nav.yml

```diff
@@ -69,6 +69,8 @@
       permalink: /guides/libraries/useful-libraries
     - title: Creating packages
       permalink: /guides/libraries/create-library-packages
+    - title: Verified publishers
```

Comment:
There's no such page... and no need, right? It seems like we can cover this in the Publishing packages page and maybe the Package reference.

Reply:
There is now; I forgot to `git add` it yesterday.

This is meant to be the description of what verified publishers are for package *consumers*. The publishing page is for publishers, and has a lot of irrelevant content for consumers.